src/main/scala/de/upb/cs/swt/delphi/instanceregistry/daos/Tables.scala

```diff
@@ -0,0 +1,43 @@
+package de.upb.cs.swt.delphi.instanceregistry.daos
+import slick.jdbc.H2Profile.api._
+import slick.sql.SqlProfile.ColumnOption.NotNull
+
+class Instances(tag: Tag) extends Table[(Long, String, Long, String, String, String, String, String)](tag, "instances") {
+  def id = column[Long]("id", O.PrimaryKey, O.AutoInc) // This is the primary key column
+  def host = column[String]("host", O.Length(50), NotNull)
```

Comment:
Please choose a higher character limit for ```host```, it may be an IPv6 address which, together with protocol and host, could exceed the limit of 50 characters.